packages/node_modules/@webex/bin-sauce-connect/src/start.js

```diff
@@ -87,7 +88,7 @@ function connect() {
     spawn('sc/bin/sc', [
       '-B', [
         'mercury-connection-intb.ciscospark.com',
-        'idbroker.webex.com',
+        'idbrokerbts.webex.com',
```

Comment:
Does this line need to be changed too?
Does this need to be `${process.env.IDBROKER_BASE_URL || 'idbroker.webex.com'}` also?

Reply:
No, we want to add here: 

```
'idbroker.webex.com',
'idbrokerbts.webex.com',
```